packages/graphql/src/schema/to-compose.ts

```diff
@@ -132,8 +131,8 @@ export function objectFieldsToSubscriptionsWhereInputFields(
                 [`${f.fieldName}_NOT_INCLUDES`]: f.typeMeta.name,
             }),
             ...(ifAnyTypeExceptArrayAndBoolean && {
-                [`${f.fieldName}_IN`]: `[${f.typeMeta.name}]`,
-                [`${f.fieldName}_NOT_IN`]: `[${f.typeMeta.name}]`,
+                [`${f.fieldName}_IN`]: `[${f.typeMeta.name}${f.typeMeta.required ? "!" : ""}]`,
+                [`${f.fieldName}_NOT_IN`]: `[${f.typeMeta.name}${f.typeMeta.required ? "!" : ""}]`,
```

Comment:
_Naughty_, this is technically a breaking change, this should really be saved until 4.0 if it can.